circle.yml

```diff
@@ -13,4 +13,5 @@ test:
     - gem install --no-rdoc brakeman; bundle exec brakeman --exit-on-warn .
     - gem install --no-rdoc ruby_audit; bundle exec ruby-audit check
     - gem install --no-rdoc bundler-audit; bundle-audit update; bundle-audit check
+    - gem install --no-rdoc dawnscanner
```

Comment:
(on the same line, separated by a semicolon! so the full line should be:

`- gem install --no-rdoc dawnscanner; bundle exec dawn ...` etc)